packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md

```diff
@@ -1 +1,5 @@
-A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
\ No newline at end of file
+A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.
```

Comment:
```suggestion
A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```

Reply:
"A CRChangeListItem is an item **in** the…"
It's a shame there's no suggestion for suggestions